packages/store/addon/-private/store-service.ts

```diff
@@ -134,20 +134,20 @@ export interface CreateRecordProperties {
   The store provides multiple ways to create new record objects. They have
   some subtle differences in their use which are detailed below:
 
-  [createRecord](../methods/createRecord?anchor=createRecord) is used for creating new
+  [createRecord](./classes/Store/methods?anchor=createRecord) is used for creating new
```

Comment:
if `./classes/Store/` is necessary for createRecord then it would be necessary for push and pushPayload as well. Probably worth booting up the docs app locally to check.

- generate the api doc data https://github.com/ember-learn/ember-jsonapi-docs
- the app: https://github.com/ember-learn/ember-api-docs/blob/master/README.md

I forget the precise magic formula for getting these things working together locally.

Reply:
Thanks.